resources/measures/upgrade_hvac_add_heat_pump_rtu/measure.rb

```diff
@@ -1989,17 +1989,25 @@ def run(model, runner, user_arguments)
       new_air_to_air_heatpump.setAvailabilitySchedule(unitary_availability_sched)
       new_air_to_air_heatpump.setDehumidificationControlType(dehumid_type)
       new_air_to_air_heatpump.setSupplyAirFanOperatingModeSchedule(supply_fan_op_sched)
-      new_air_to_air_heatpump.setControlType('Load') ##cc-tmp
+      new_air_to_air_heatpump.setControlType('Load') ## cc-tmp
       new_air_to_air_heatpump.setName("#{thermal_zone.name} RTU SZ-VAV Heat Pump")
       new_air_to_air_heatpump.setMaximumSupplyAirTemperature(50)
       new_air_to_air_heatpump.setDXHeatingCoilSizingRatio(1+performance_oversizing_factor)
-      # set cooling design flow rate
-      new_air_to_air_heatpump.setSupplyAirFlowRateDuringCoolingOperation(hash_clg_airflow_stgs[4])
-      # set heating design flow rate
-      new_air_to_air_heatpump.setSupplyAirFlowRateDuringHeatingOperation(hash_htg_airflow_stgs[4])
-      # set no load design flow rate
-      new_air_to_air_heatpump.resetSupplyAirFlowRateMethodWhenNoCoolingorHeatingisRequired
-      new_air_to_air_heatpump.setSupplyAirFlowRateWhenNoCoolingorHeatingisRequired(min_airflow_m3_per_s)
+
+      # handle deprecated methods for OS Version 3.7.0
+      if model.version < OpenStudio::VersionString.new('3.7.0')
+        # set cooling design flow rate
+        new_air_to_air_heatpump.setSupplyAirFlowRateDuringCoolingOperation(hash_clg_airflow_stgs[4])
+        # set heating design flow rate
+        new_air_to_air_heatpump.setSupplyAirFlowRateDuringHeatingOperation(hash_htg_airflow_stgs[4])
+        # set no load design flow rate
+        new_air_to_air_heatpump.resetSupplyAirFlowRateMethodWhenNoCoolingorHeatingisRequired
+        new_air_to_air_heatpump.setSupplyAirFlowRateWhenNoCoolingorHeatingisRequired(min_airflow_m3_per_s)
+      else
+        new_air_to_air_heatpump.autosizeSupplyAirFlowRateDuringCoolingOperation
+        new_air_to_air_heatpump.autosizeSupplyAirFlowRateDuringHeatingOperation
+        new_air_to_air_heatpump.autosizeSupplyAirFlowRateWhenNoCoolingorHeatingisRequired
+      end
```

Comment:
I don't think these are correct changes. Originally the measure was setting the flow rates, and this will change that behavior to autosize the flow rates. Only `resetSupplyAirFlowRateWhenNoCoolingorHeatingisRequired` is deprecated, but that doesn't seem necessary when the 'No Cooling or Heating' flow rate is set right after. Just remove the line with that method.

Reply:
Good point I relized my error after going through these. Comments. I think I have it handled correctly now.